.github/workflows/create-release.yml

```diff
@@ -2,7 +2,7 @@ name: Create Release
 
 on:
   schedule:
-  - cron: '*/30 * * * *'  # every 30 minutes
```

Comment:
I agree with this change, I think its good to run it every half an hour as the rest of the stacks do.

Reply:
This change is based on https://github.com/paketo-buildpacks/github-config/commit/06cb82f306fd58c388eef9bf68114fe55df517ba. Which option is the correct one?

Reply:
oh, I see, I agree with this one then. Probably they changed that due to too many notifications. thanks!